SD-K9/src/environments/environment.ts

```diff
@@ -4,7 +4,17 @@
 
 export const environment = {
   production: false,
-  apiKey: 'AIzaSyA_u2fkanThpKMP4XxqLVfT9uK0puEfRns'
+  apiKey: 'AIzaSyA_u2fkanThpKMP4XxqLVfT9uK0puEfRns',
```

Comment:
Same as last comment, should this be private?

Reply:
Explained in your last comment, however this specific apiKey you are refencing is the Google Maps API key.